tier4_system_msgs/msg/MRMBehaviorStatus.msg

```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time stamp
+bool is_available
+bool is_operating
```

Comment:
Do you mean you want to get these statuses independently?

Could you show me some use cases that the following definition can't support?

```
NOT_AVAILABLE=0
AVAILABLE=1
OPERATING=1

int8 state
```

Reply:
There is no situation that is (!is_available && is_operating), so change as above.

Reply:
Fix the state in the following commit.
https://github.com/tier4/tier4_autoware_msgs/pull/60/commits/64e7f6008983144ccb5a3905a4dc44ae48cf425d